tutorials/dataframe/df022_createKahan.C

```diff
@@ -0,0 +1,98 @@
+/// \file
+/// \ingroup tutorial_dataframe
+/// \notebook
+/// This tutorial shows how to implement a Kahan summation custom action.
+///
+/// \macro_code
+///
+/// \date July 2018
+/// \author Enrico Guiraud, Danilo Piparo, Massimo Tumolo
+
+class KahanSum : public ROOT::Detail::RDF::RActionImpl<class KahanSum> {
```

Comment:
I would template on the FP type (float or double) and propagate accordingly to the class. For example, I would also add a static assert that tells the user if she requested this summation on integer types.
See http://www.cplusplus.com/reference/type_traits/is_floating_point.